### ISSUE TITLE
WCAG 2.2（2024年12月版）への更新

### ISSUE THREAD

**Issue by momdo** [waic/wcag22#360]
- 対象URL
  - https://www.w3.org/TR/2024/REC-WCAG22-20241212/
- Change Log
  - https://www.w3.org/TR/2024/REC-WCAG22-20241212/#changelog
- 前の勧告との差分
  - https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F

一見、日本語訳としては更新が少なそうだが、HTMLが変更されている箇所がある。流し込み直した方が安全か？

## 日本語訳として更新が必要な箇所

- [Abstract](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#abstract)
- [Success Criterion 2.4.11 Focus Not Obscured (Minimum)](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#focus-not-obscured-minimum)
- [Success Criterion 2.4.13 Focus Appearance](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#focus-appearance)
- 用語集：encloses（削除されている）
- [motion animation](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#dfn-motion-animation)
- [programmatically determined link context](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#dfn-programmatically-determined-link-context)
- [single pointer](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#dfn-single-pointer)
- [used in an unusual or restricted way](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#dfn-used-in-an-unusual-or-restricted-way)
- [7. Input Purposes for User Interface Components](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#input-purposes)の`postal-code`
- [A. Change Log](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2023%2FREC-WCAG22-20231005%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2024%2FREC-WCAG22-20241212%2F#changelog)

**Comment by caztcha** [comment]
Slack (#wg4-wcag22) で方針確認。

- 基本的には、コンテンツのテキスト部分以外は本家に揃えるというポリシーで行きたい
- CSSなどに差分があることはあまり望ましくない

ということで、「流し込み直し」をしたうえで、翻訳します。以下の手順で進めます。

1. GitHub 上にある WCAG 2.2 本体 (index.html) について、別ブランチを切って、WCAG 2.2 の最新版 (2024-12-12) の HTML コードをコピペする (コンテンツのテキスト部分はいったん英語になる）
2. 2023-10-05 時点ですでに日本語訳した文面も含めて、最新版 (2024-12-12) の HTML にひとつひとつ移植してゆく。

**Comment by caztcha** [comment]
PR #374 で対応します。

**Comment by momdo** [comment]
マージ済みなのでCloseします
